api-tests/core/admin/ee/review-workflows.test.api.js

```diff
@@ -180,6 +180,43 @@ describeOnCondition(edition === 'EE')('Review workflows', () => {
     });
   });
 
+  describe('Create workflow', () => {
+    test('You can not create a workflow without stages', async () => {
+      const res = await requests.admin.post('/admin/review-workflows/workflows', {
+        body: {
+          name: 'testWorkflow',
+        },
+      });
+
+      if (hasRW) {
+        expect(res.status).toBe(400);
+        expect(res.body.error.message).toBe('Can not create a workflow without stages');
```

Comment:
```
    Expected: "Can not create a workflow without stages"
    Received: "stages is a required field"

```

I wonder if we should remove the .required() from `packages/core/admin/ee/server/validation/review-workflows.js` ?

Reply:
I think we should leave the .required(), keeping it is necessary because .min() alone doesn't ensure that the attribute is not null.

Reply:
What I could change is the error message so its consistent 🤔 wdyt?